spec/views/tracks/new.html.erb_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe "tracks/new", type: :view do
+  before(:each) do
+    assign(:track, Track.new(
+      :name => "MyString",
```

Comment:
Indent the first parameter one step more than `Track.new(`.<br>Use the new Ruby 1.9 hash syntax.